### PR TITLE
add edge condition unit test for batch write

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

**Comment by marsishandsome** [comment]
@zhexuany PTAL

**Comment by marsishandsome** [comment]
@zhexuany PTAL

File: tikv-client/src/main/java/com/pingcap/tikv/TwoPhaseCommitter.java [hunk 1]
```diff
@@ -288,6 +288,10 @@ private void doPrewriteSecondaryKeysInBatches(
         Kvrpcpb.Mutation mutation =
             Kvrpcpb.Mutation.newBuilder().setKey(key).setValue(value).setOp(Kvrpcpb.Op.Put).build();
         mutations.put(key, mutation);
+      } else {
```
Comment (zhexuany) [inline tikv-client/src/main/java/com/pingcap/tikv/TwoPhaseCommitter.java]: Should we explain when value can be nil?

Comment (marsishandsome) [reply tikv-client/src/main/java/com/pingcap/tikv/TwoPhaseCommitter.java]: done
